configuration/multimedia

```diff
@@ -0,0 +1,160 @@
+---
+layout: doc
+title: Multimedia
+permalink: /doc/multimedia/
+redirect_from:
+- /en/doc/multimedia/
+- /doc/Multimedia/
+- /wiki/Multimedia/
+---
+
+Multimedia
+==========
+
+In order to choose Qubes OS as your primary OS it must be able to do all tasks, including playing multimedia content.
```

Comment:
Same comment as from @andrewdavidwong. "all tasks" is much more than "all tasks you do". And even that isn't strictly necessary, because you may not necessary do all the tasks on the same ("primary") OS - for example you may have another one for gaming.